embulk-api/src/test/java/org/embulk/spi/json/TestJsonString.java

```diff
@@ -140,6 +140,15 @@ public void testEscape() {
         assertFalse(string.equals(FakeJsonString.of(string.getString())));
     }
 
+    @SuppressWarnings("checkstyle:IllegalTokenText")
+    @Test
+    public void testExample() {
```

Comment:
Minor: It seems reasonable the name `testLiteral` or something instead of `testExample`.

Reply:
Thanks for your comment. Actually, these cases are "examples" contained in the EEP Draft: #1538, then the `Example` intended that.